### DISCUSSION TITLE
Disable resize for some shape

### DISCUSSION THREAD

**Discussion by aheadweb** [pikasojs/pikaso#92] [Q&A] [answered]
Hello, is there a way to change the resizing ability for a specific shape or group.

I can do this for all objects like this `editor.current.selection.transformer.resizeEnabled(false)`
And I would like something like this
`const shape  = editor.current.shapes.line.insert(...);`
`shape[smth].resizeEnabled(false)`
or something similar for my custom objects
`const arrow = editor.current.shapes.bezierArrow.insert(...);`
`arrow[smth].resizeEnabled(false)`

**Comment by raminious** [comment, marked as answer]
```ts
const myRect = editor.shapes.rect.insert({
  x: 200,
  y: 200,
  width: 100,
  height: 80,
  fill: "blue"
});

myRect.config = {
  ...myRect.config,
  transformer: {
    enabledAnchors: []
  }
};

```

Working Example:   

https://codesandbox.io/s/pikaso-selectable-example-t1pnig

**Reply by aheadweb** [reply]
Thank you. If someone needs it, this can be done in custom objects
```
export class MyModel extends ShapeModel<Konva.Arc, Konva.ArcConfig> {
  constructor(board: Board, node: Konva.Arc) {
    super(board, node);
    this.config = {
      ...this.config,
      transformer: {
        enabledAnchors: [],
      },
    };
  }

  public get type() {
    return "myType";
  }
}
```
